templates/items/shield.html

```diff
@@ -22,7 +22,7 @@ <h4 class="item-type">{{itemType}}</h4>
         {{!-- Description --}}
         <div class="tab flexrow active" data-group="primary" data-tab="description">
             <p>Description</p>
-            {{editor content=data.description.value target="data.description.value" button=true owner=owner editable=editable}}
+            {{editor content=data.description target="data.description" button=true owner=owner editable=editable}}
```

Comment:
I think there is quite some duplication here. Isn't there a way to create a part "description" that you would reuse every time that you want to put a description? It could go in a file in templates/parts.

Reply:
Definitely, we can make a part for the description to share it among the entities